backend/src/organizer/organizer.service.ts

```diff
@@ -1,19 +1,36 @@
-import { Injectable, NotFoundException } from '@nestjs/common';
+import {
+  BadRequestException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
 import { Organizer } from '@prisma/client';
-import { NotFoundError } from '@prisma/client/runtime';
+import {
+  NotFoundError,
+  PrismaClientKnownRequestError,
+} from '@prisma/client/runtime';
 import { PrismaService } from '../global/prisma/prisma.service';
 import { OrganizerDTO, TypeEnum } from './dto/organizer.dto';
 @Injectable()
 export class OrganizerService {
   constructor(private readonly prisma: PrismaService) {}
 
-  public async createNewOrganizer(
-    organizerDTO: OrganizerDTO
-  ): Promise<OrganizerDTO> {
-    const newOrganizer = await this.prisma.organizer.create({
-      data: organizerDTO,
+  public async createNewOrganizer(organizerDTO: OrganizerDTO): Promise<any> {
+    const result = await this.prisma.organizer.findMany({
+      where: {
+        name: {
+          equals: organizerDTO.name,
+          mode: 'insensitive',
+        },
+      },
     });
-    return OrganizerService.convertToDTO(newOrganizer);
+    if (result.length > 0) {
```

Comment:
Used if else here to check if the `result` variable will return an array containing the data that matches the query if none, it will proceed with the creation of organizer.